app/views/logs/_log_list.html.erb

```diff
@@ -11,12 +11,15 @@
 </div>
 <div class="govuk-grid-column-one-quarter govuk-!-text-align-right">
   <% if logs&.any? && (display_delete_logs?(@current_user, searched, filter_type) || in_organisations_tab?) %>
-  <%# remove @ on current user %>
     <%= govuk_link_to "Delete logs", delete_logs_path, class: "app-!-colour-red" %>
   <% end %>
 </div>
 </div>
 </h2>
 <% logs.map do |log| %>
-  <%= render(LogSummaryComponent.new(current_user:, log:)) %>
+  <% if log.sales? %>
+    <%= render(SalesLogSummaryComponent.new(current_user:, log:)) %>
+    <% else %>
+    <%= render(LettingsLogSummaryComponent.new(current_user:, log:)) %>
```

Comment:
very minor point, but render is one place where it's most common not to use parentheses around arguments in the method call in rails
```suggestion
    <%= render LettingsLogSummaryComponent.new(current_user:, log:) %>
```